CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Usage:
 * Fix #1142: Log informative message whenever a goal/task is skipped
 * Fix #1197: Kubernetes Gradle Build tasks don't account for image model configuration skip field
 * Fix #1245: Deprecate `maven.jkube.io` annotation prefix used in enrichers in favor of `jkube.eclipse.org`
+* Fix #1244: Helm Parameters/Variables/Values.yaml can be configured via XML/DSL
```

Comment:
is it worth adding "feat: Helm supports placeholders/variables with dotted notation" in the changelog ?

Reply:
Yes, probably